cmd/dp-deployer/main.go

```diff
@@ -49,32 +50,31 @@ func main() {
 
 	log.Event(ctx, "config on startup", log.INFO, log.Data{"config": cfg})
```

Comment:
Consider removing the `healthcheckConfig` struct - it does not appear to be used.